skactiveml/pool/regression/_query_by_committee.py

```diff
@@ -0,0 +1,181 @@
+        sample_weight: array-like of shape (n_samples), optional (default=None)
+            Weights of training samples in `X`.
+        candidates : None or array-like of shape (n_candidates), dtype=int or
+            array-like of shape (n_candidates, n_features),
+            optional (default=None)
+            If candidates is None, the unlabeled samples from (X,y) are
+            considered as candidates.
+            If candidates is of shape (n_candidates) and of type int,
+            candidates is considered as the indices of the samples in (X,y).
+            If candidates is of shape (n_candidates, n_features), the
+            candidates are directly given in candidates (not necessarily
+            contained in X).
+        batch_size : int, optional (default=1)
+            The number of samples to be selected in one AL cycle.
+        return_utilities : bool, optional (default=False)
+            If true, also return the utilities based on the query strategy.
+
+        Returns
+        -------
+        query_indices : numpy.ndarray of shape (batch_size)
+            The query_indices indicate for which candidate sample a label is
+            to queried, e.g., `query_indices[0]` indicates the first selected
+            sample.
+            If candidates is None or of shape (n_candidates), the indexing
+            refers to samples in X.
+            If candidates is of shape (n_candidates, n_features), the indexing
+            refers to samples in candidates.
+        utilities : numpy.ndarray of shape (batch_size, n_samples) or
+            numpy.ndarray of shape (batch_size, n_candidates)
+            The utilities of samples after each selected sample of the batch,
+            e.g., `utilities[0]` indicates the utilities used for selecting
+            the first sample (with index `query_indices[0]`) of the batch.
+            Utilities for labeled samples will be set to np.nan.
+            If candidates is None or of shape (n_candidates), the indexing
+            refers to samples in X.
+            If candidates is of shape (n_candidates, n_features), the indexing
+            refers to samples in candidates.
+        """
+        X, y, candidates, batch_size, return_utilities = self._validate_data(
+            X, y, candidates, batch_size, return_utilities, reset=True
+        )
+
+        check_type(fit_ensemble, "fit_ensemble", bool)
+
+        if isinstance(ensemble, SkactivemlRegressor) and hasattr(
+            ensemble, "n_estimators"
+        ):
+            if fit_ensemble:
+                ensemble = clone(ensemble).fit(X, y, sample_weight)
+            est_arr = ensemble.estimators_
+        elif _is_arraylike(ensemble):
+            est_arr = deepcopy(ensemble)
+            for idx, est in enumerate(est_arr):
+                check_type(est, f"ensemble[{idx}]", SkactivemlRegressor)
+                if fit_ensemble:
+                    est_arr[idx] = est.fit(X, y, sample_weight)
```

Comment:
`k_bootstraps` does not affect the ensemble here. Maybe update the documentation for `k_bootstraps` and what it influences?